com.unity.render-pipelines.universal/Runtime/SceneViewDrawMode.cs

```diff
@@ -10,7 +10,8 @@ internal static class SceneViewDrawMode
 
         static bool RejectDrawMode(SceneView.CameraMode cameraMode)
         {
-            if (cameraMode.drawMode == DrawCameraMode.ShadowCascades ||
+            if (cameraMode.drawMode == DrawCameraMode.Wireframe ||
```

Comment:
Wireframe mode now works, we should not add it to reject mode anymore

Reply:
Removed wireframe mode from this test.